docs/UserGuide/Cluster/Cluster-Setup.md

```diff
@@ -143,284 +134,212 @@ Please set the important parameters in iotdb-datanode.properties:
+For more details about other configuration parameters of ConfigNode, see the
+[ConfigNode Configurations](https://iotdb.apache.org/UserGuide/Master/Reference/ConfigNode-Config-Manual.html).
 
-The stop-standalone.sh may not work well without sudo, since IoTDB's port numbers may be invisible without permission. 
-If stop-standalone.sh meets some error, you can use "jps" or "ps aux | grep iotdb" to obtain the process ids,
-and use "sudo kill -9 <process-id>" to manually stop the processes.
+### 6.1.2 Add ConfigNode (Optional)
 
-## Start Cli
+You can add more ConfigNodes to the cluster to ensure high availability of ConfigNodes.
+Ensure that all configuration parameters in the ./conf/iotdb-common.properites are the same as those in the Seed-ConfigNode; 
+otherwise, it may fail to start or generate runtime errors.
+Open the configuration file ./conf/iotdb-confignode.properties for the new ConfigNode and set the following parameters:
 
-Cli shell is in sbin folder.
+| **Configuration**              | **Description**                                                                                                                                            |
+|--------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cn\_internal\_address          | Internal rpc service address of ConfigNode                                                                                                                 |
+| cn\_internal\_port             | Internal rpc service port of ConfigNode                                                                                                                    |
+| cn\_consensus\_port            | ConfigNode replication consensus protocol communication port                                                                                               |
+| cn\_target\_config\_node\_list | Configure the internal communication address of an arbitrary running ConfigNode. The internal communication address of the seed ConfigNode is recommended. |
 
-Start on Linux:
-```
-./datanode/sbin/start-cli.sh
+After the configuration is complete, run the startup script:
 ```
+# Linux foreground
+bash ./sbin/start-confignode.sh
 
-Start on Windows:
-```
-datanode\sbin\start-cli.bat
+# Linux background
+nohup bash ./sbin/start-confignode.sh >/dev/null 2>&1 &
+
+# Windows
+.\sbin\start-confignode.bat
 ```
 
-## Shrink the Cluster
+For more details about other configuration parameters of ConfigNode, see the
+[ConfigNode Configurations](https://iotdb.apache.org/UserGuide/Master/Reference/ConfigNode-Config-Manual.html).
 
-### Remove ConfigNode
+### 6.1.3 Start DataNode
 
-Execute the remove-confignode shell on an active ConfigNode, and make sure that there is at least one active ConfigNode in Cluster after removing.
+You can add any number of DataNodes to the cluster.
+Before adding a new DataNode, 
```

Comment:
The same issue. All settings are done! 
You just need to remind users please read Chap 5.2.3 if he/she has not modify the configuration.

Reply:
Fixed~